app/views/application/equality-monitoring/disabilities.html

```diff
@@ -13,81 +13,75 @@
     {{ govukInput({
       classes: "govuk-!-width-two-thirds",
       label: {
-        text: "Describe your disability (optional)"
+        text: "Your disability or health condition (optional)"
       }
     } | decorateApplicationAttributes(["equality-monitoring", "disabilities-other"])) }}
   {% endset -%}
 
+  <p class="govuk-body">Training providers will not see your answer to this question when deciding whether to offer you a place.</p>
+
+  <p class="govuk-body">They will only see your answer if you accept an offer from them.</p>
+
+  <!-- IF they haven’t put anything in the Ask for support if you’re disabled field -->
+  <p class="govuk-body">
+You can <a href="/application/{{ applicationId }}/additional-support" class="govuk-link">ask for support as part of your application</a> if you want to tell training providers about your needs straight away.    </p>
+  <!-- end if -->
+
+
   {{ govukCheckboxes({
     fieldset: {
+      legend: {
+        text: "Do you have any of these disabilities or health conditions?",
+        classes: "govuk-fieldset__legend--m"
+      },
       attributes: {
         "data-module": "clear-hidden"
       }
     },
     items: [{
-      text: "Blind",
-      label: {
-        classes: "govuk-label--s"
-      },
-      hint: {
-        text: "Or a serious visual impairment which is not corrected by glasses"
-      }
+      text: "Blindness or a visual impairment not corrected by glasses"
     }, {
-      text: "Deaf",
-      label: {
-        classes: "govuk-label--s"
-      },
-      hint: {
-        text: "Or a serious hearing impairment"
-      }
+      text: "Deafness or a serious hearing impairment"
     }, {
-      text: "Learning difficulty",
-      label: {
-        classes: "govuk-label--s"
-      },
-      hint: {
-        text: "For example, dyslexia, dyspraxia or ADHD"
-      }
+      text: "Dyslexia, dyspraxia or attention deficit hyperactivity disorder (ADHD)"
     }, {
-      text: "Long-standing illness",
-      label: {
-        classes: "govuk-label--s"
-      },
+      text: "Long-term illness",
       hint: {
         text: "For example, cancer, HIV, diabetes, chronic heart disease or epilepsy"
       }
     }, {
       text: "Mental health condition",
-      label: {
-        classes: "govuk-label--s"
-      },
       hint: {
         text: "For example, depression, schizophrenia or anxiety disorder"
       }
     }, {
       text: "Physical disability or mobility issue",
-      label: {
-        classes: "govuk-label--s"
-      },
       hint: {
         text: "For example, impaired use of arms or legs, use of a wheelchair or crutches"
       }
     }, {
-      text: "Social or communication impairment",
-      label: {
-        classes: "govuk-label--s"
-      },
-      hint: {
-        text: "For example Asperger’s, or another autistic spectrum disorder"
-      }
+      text: "Autistic spectrum condition or a speech and language impairment"
```

Comment:
We could change to "Autistic spectrum condition or another social or communication condition" perhaps, however this loses the specificity of "speech and language impairment"?

Alternatively to be fully comprehensive we could say "Autistic spectrum condition, a speech or language impairment, or another social or communication condition".

(Intention is to front-load the examples rather than the general category, to improve the ease of scanning)